generators/base-application/priorities.cjs

```diff
@@ -0,0 +1,138 @@
+const { PRIORITY_PREFIX, QUEUE_PREFIX, PRIORITY_NAMES, QUEUES } = require('../base/priorities.cjs');
```

Comment:
Isn't it worth it to create a `base-platform/priorities.cjs` (a bunch of application), a `base-entity/priorities.cjs` and maybe a `base-field/priorities.cjs`?

Reply:
> Isn't it worth it to create a `base-platform/priorities.cjs` (a bunch of application).

Yes, I think we should call it `base-workspaces`.
I will create a feature request.

> a `base-entity/priorities.cjs` and maybe a `base-field/priorities.cjs`?

I don't think so.
But I think we should implement a base-incremental-application for entity changes like liquibase and maybe entity removal.

Reply:
Some users are complaining that generation is all or nothing. Splitting metamodels (and models) could ease to generate only entities and fields that have been updated

Reply:
@Tcharl the application, entities, fields and relationships are dependent on each other, it's not possible to create a consistent application without preparing everything.
See https://github.com/jhipster/generator-jhipster/issues/19989.